python/ray/serve/http_state.py

```diff
@@ -123,7 +124,9 @@ def _start_proxies_if_needed(self) -> None:
                     max_concurrency=ASYNC_CONCURRENCY,
                     max_restarts=-1,
                     max_task_retries=-1,
-                    resources={node_resource: 0.01},
+                    scheduling_strategy=NodeAffinitySchedulingStrategy(
```

Comment:
It's great that the change is so simple.

Looks like `node_resource` is now unused?